src/ai/ai.test.js

```diff
@@ -84,7 +84,7 @@ const enumerate = (G, ctx, playerID) => {
 
 describe('Step', () => {
   test('advances game state', async () => {
-    const client = Client({
+    const client = new Client({
```

Comment:
I would prefer to keep the external interface a function (rather than a constructor).